gencerts.bat

```diff
@@ -0,0 +1,16 @@
+SETLOCAL
```

Comment:
we might just point people how to generate certs in the readme so that we don't keep a bat and shell script around.

Reply:
most likely